assets/src/containers/WithRouter.js

```diff
@@ -1,26 +1,24 @@
 import React from 'react'
-import { useLocation, useNavigate, useParams } from "react-router-dom"
+import { useLocation, useNavigate, useParams } from 'react-router-dom'
 
 /**
  * This is a workaround for the fact that React Router v6 no longer has a withRouter function.
  * See https://reactrouter.com/en/main/start/faq#what-happened-to-withrouter-i-need-it
- */ 
+ */
 
-function withRouter(Component) {
-  function ComponentWithRouterProp(props) {
-    let location = useLocation()
-    let navigate = useNavigate()
-    let params = useParams()
+function withRouter (Component) {
+  function ComponentWithRouterProp (props) {
     return (
       <Component
-        		{...props}
-        		location={location}
-        		params={params}
-        		navigate={navigate}
-      		/>
+        {...props}
+        location={useLocation()}
+        params={useParams()}
+        navigate={useNavigate()}
+      />
     )
   }
 
   return ComponentWithRouterProp
 }
-export default withRouter
\ No newline at end of file
+
+export default withRouter
```

Comment:
V6 of the react-router-dom removed withRouter, so I have to write this HOC as per the documentation guide, linked in the page